sub-packages/bionemo-esm2/src/bionemo/esm2/model/finetune/dataset.py

```diff
@@ -71,13 +74,15 @@ def __init__(
     def from_csv(
         cls,
         csv_path: str | os.PathLike,
+        task_type: Literal["classification", "regression", None] = None,
         tokenizer: tokenizer.BioNeMoESMTokenizer = tokenizer.get_tokenizer(),
         ignore_labels: bool = False,
     ):
         """Class method to create a ProteinDataset instance from a CSV file.
 
         Args:
             csv_path: path to CSV file containing sequences and optionally labels column.
+            task_type (str, optional): Fine-tuning task type. Defaults to None.
```

Comment:
What does `None` mean though? Doesn't that just cause a failure? Why not just make the default one of classification / regression?

Reply:
If this is a required argument, then don't provide a default, just make it required

Reply:
`None` is used when we do inference and there is no label in the dataset

Reply:
We use `InMemoryProteinDataset` in inference and the other two datasets for finetuning, depending on the label types (single-value, per-token-value)